collection/rancher/v2.x/usage-collector/rancher2_usage_collector.sh

```diff
@@ -1,17 +1,17 @@
 #!/bin/bash
 
 echo "Report run on `date`"
-echo ""
+echo
 
 # Get docker id for rancher single node install
 DOCKER_ID=$(docker ps | grep "rancher/rancher:" | cut -d' ' -f1)
```

Comment:
Thoughts on using something like `docker ps -q --filter ancestor=rancher/rancher` to cover single/HA in one command?